src/components/10-atoms/checkbox/react/story.jsx

```diff
@@ -33,18 +34,25 @@ storiesOf('Examples/Checkbox/React', module)
     }
 
     ReactDOM.render(
-      <AXACheckboxReact
-        name={name}
-        label={label}
-        disabled={disabled}
-        variant={variant}
-        required={required}
-        styled={styled}
-        onChange={e =>
-          console.log('checkbox', name, ' changed to: ', e.target.checked)
-        }
-        error={errortext ? 'Please accept our terms and conditions.' : ''}
-      />,
+      <>
```

Comment:
?

Reply:
https://medium.com/@rossbulat/the-empty-tag-is-shorthand-for-react-fragment-6129e70e418a